### DISCUSSION TITLE
Dodanie "jednostki" do GET /order/checkout-forms*

### DISCUSSION THREAD

**Discussion by arekpietrzyk** [allegro/allegro-api#8768] [Suggestions - rejected]
W nawiązaniu do niezmiernie usprawniającej post-processing zamówień zmiany wprowadzającej stawkę VAT, chciałbym zasugerować niewiele mniej istotną wartość uzupełniającą, jednostkę.

Obok "quantity" ucieszyłaby mnie - i zapewne wielu innych - wartość "unit" pochodząca ze zdefiniowanej podczas tworzenia oferty wartości "stock.unit".

Zmiana oczekiwana w ramach odpowiedzi z następujących endpointów:

GET /order/checkout-forms
GET /order/checkout-forms/{id}

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Obecnie jednak nie planujemy wprowadzać takiej zmiany, dlatego aktualizuję status sugestii na "odrzucona".
